sys/shell/commands/sc_openwsn.c

```diff
@@ -0,0 +1,569 @@
+    puts(
+        "\tcell rmv <slot_offset> <channel_offset> <adv|tx|rx> [<address>]: remove cell directly in schedule ");
+}
+
+static int _cell_list_cmd(char *arg)
+{
+    (void)arg;
+
+    for (int i = 0; i < MAXACTIVESLOTS; i++) {
+        switch (schedule_vars.scheduleBuf[i].type) {
+            case CELLTYPE_TX:
+                printf("neigh: %s, slot: %03i, chan: %02i, type: TX\n", \
+                       netif_addr_to_str(
+                           schedule_vars.scheduleBuf[i].neighbor.addr_64b,
+                           8, _addr_str),
+                       schedule_vars.scheduleBuf[i].slotOffset, \
+                       schedule_vars.scheduleBuf[i].channelOffset);
+                break;
+            case CELLTYPE_RX:
+                printf("slot: %03i, chan: %02i, type: RX\n", \
+                       schedule_vars.scheduleBuf[i].slotOffset, \
+                       schedule_vars.scheduleBuf[i].channelOffset);
+                break;
+            case CELLTYPE_TXRX:
+                printf("neigh: %s, slot: %03i, chan: %02i, type: RXTX\n", \
+                       netif_addr_to_str(
+                           schedule_vars.scheduleBuf[i].neighbor.addr_64b,
+                           8, _addr_str),
+                       schedule_vars.scheduleBuf[i].slotOffset, \
+                       schedule_vars.scheduleBuf[i].channelOffset);
+                break;
+            default:
+                break;
+        }
+    }
+
+    return 0;
+}
+
+static int _cell_manage_cmd(int argc, char **argv)
+{
+    open_addr_t addr;
+
+    memset(&addr, 0, sizeof(addr));
+    cellType_t type;
+    int res;
+    bool add;
+
+    if (argc < 5) {
+        _print_cell_usage();
+        return -1;
+    }
+
+    if (argc == 6) {
+        addr.type = ADDR_64B;
+        size_t len = netif_addr_from_str(argv[5], addr.addr_64b);
+        if (len == 0) {
+            puts("Error: invalid address");
+            return -1;
+        }
+    }
+    else {
+        addr.type = ADDR_ANYCAST;
+    }
+
+    add = strcmp(argv[1], "add") == 0 ? true : false;
+
+    if (strcmp(argv[4], "adv") == 0) {
+        type = CELLTYPE_TXRX;
+    }
+    else if (strcmp(argv[4], "tx") == 0) {
+        type = CELLTYPE_TX;
+    }
+    else if (strcmp(argv[4], "rx") == 0) {
+        type = CELLTYPE_RX;
+    }
+    else {
+        _print_cell_usage();
+        return -1;
+    }
+
+    if (add) {
+        res = schedule_addActiveSlot(atoi(argv[2]), type, true, false,
+                                     atoi(argv[3]), &addr);
+    }
+    else {
+        res = schedule_removeActiveSlot(atoi(argv[2]), type, true,
+                                        &addr);
+    }
+
+    if (res == 0) {
+        puts("Successfully set link");
+        return 0;
+    }
+    else {
+        puts("Something went wrong (duplicate link?)");
+        return -1;
+    }
+}
+
+static int _cell_cmd(int argc, char **argv)
+{
+    if (argc < 2) {
+        _print_cell_usage();
+        return 1;
```

Comment:
```suggestion
        return -1;
```
In other places, `-1` is returned in case of an error. Maybe use this error value as well here ?